beamer/tests/agent/test_request.py

```diff
@@ -363,20 +363,6 @@ def test_agent_only_claim_once_after_restart(request_manager, token, agent, dire
                     break
 
 
-def test_event_fetcher_progress_without_events(agent, direction):
```

Comment:
this test should still pass IMO

Reply:
actually this test is normally looking for update block number without any new events, and now block number shouldnt increase normally

Reply:
it should, Even though there aren't new events, new blocks should be updated in fetch()